db/migrate/20170321171136_add_uri_extra_to_items.rb

```diff
@@ -0,0 +1,5 @@
+class AddUriExtraToItems < ActiveRecord::Migration[5.0]
+  def change
+    add_column :items, :uri_extra, :string
```

Comment:
Hi @pachonk ,

Minor thing. Could you please rename `uri_extra` to `uri_fragment`? It more accurately represents what it is. Otherwise, it looks good. Thanks!